packages/extra/seaweedfs/templates/external/cosi-cluster-role.yaml

```
@@ -0,0 +1,59 @@
+- apiGroups: ["coordination.k8s.io"]
+  resources: ["leases"]
+  verbs:
+    - "get"
+    - "watch"
+    - "list"
+    - "delete"
+    - "update"
+    - "create"
+- apiGroups: [""]
+  resources:
+    - "secrets"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)